docs/community.md

```diff
@@ -10,8 +10,7 @@ Games submitted by the MakeCode community.
         "name": "Space Fight",
         "description": "Compete with a friend for the high score in a space adventure, by Alex Kulcsar",
```

Comment:
Maybe we should use the forum user names "AlexK" instead. What do you think?

Reply:
I don't really have a strong opinion, slight preference towards using actual names for attribution just cause it feels somewhat official, but will defer to you if you think username is better?